data/change-streams/README.rst

```diff
@@ -149,6 +149,20 @@ Although synchronous drivers must provide a `non-blocking mode of iteration <../
 
 If the test expects an error and one was not thrown by either creating the change stream or executing the test's operations, iterating the change stream once allows for an error to be thrown by a ``getMore`` command. If the test does not expect any error, the change stream should be iterated only until it returns as many result documents as are expected by the test.
 
+Testing on Sharded Clusters
```

Comment:
Assuming we already do all of this.

Reply:
Yeah, this prose was added for SPEC-1506 because some drivers were failing tests on sharded clusters when doing something like:

```
changeStream = collection.watch([])
collection.insert({x: 1})
event = changeStream.next()
assertNotEmpty(event)
```

Go's `Next` function blocks and retries `getMore`'s until it can get a non-empty response or an error, so this did not affect us.